app/src/main/java/org/p2p/wallet/home/ui/crypto/bottomsheet/TokenVisibilityChangeBottomSheet.kt

```diff
@@ -0,0 +1,66 @@
+package org.p2p.wallet.home.ui.crypto.bottomsheet
+
+import androidx.core.os.bundleOf
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.setFragmentResult
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.p2p.core.token.Token
+import org.p2p.wallet.R
+import org.p2p.wallet.common.ui.bottomsheet.BaseDoneBottomSheet
+import org.p2p.wallet.databinding.DialogTokensVisibilityChangePartBinding
+import org.p2p.wallet.utils.args
+import org.p2p.wallet.utils.withArgs
+
+private const val ARG_TOKEN = "ARG_TOKEN"
+private const val ARG_TOKEN_VISIBILITY_STATE = "ARG_TOKEN_VISIBILITY_STATE"
+
+class TokenVisibilityChangeBottomSheet : BaseDoneBottomSheet() {
+
+    companion object {
+        fun show(
+            fm: FragmentManager,
+            token: Token.Active,
+            isTokenHidden: Boolean,
+            requestKey: String = ARG_REQUEST_KEY,
+            resultKey: String = ARG_RESULT_KEY
+        ) = TokenVisibilityChangeBottomSheet().withArgs(
+            ARG_TOKEN to token,
+            ARG_TOKEN_VISIBILITY_STATE to isTokenHidden,
+            ARG_REQUEST_KEY to requestKey,
+            ARG_RESULT_KEY to resultKey
+        ).show(fm, TokenVisibilityChangeBottomSheet::javaClass.name)
```

Comment:
explicit return type plus new lines
```suggestion
        ) = TokenVisibilityChangeBottomSheet().withArgs(
            ARG_TOKEN to token,
            ARG_TOKEN_VISIBILITY_STATE to isTokenHidden,
            ARG_REQUEST_KEY to requestKey,
            ARG_RESULT_KEY to resultKey
        )
            .show(fm, TokenVisibilityChangeBottomSheet::javaClass.name)
```